### PR TITLE
Ui stuff

### PR DESCRIPTION
**Pull request by cjacobwade**

### CODE REVIEW

File: Assets/Scripts/Managers/DayCycleManager.cs [hunk 1]
```diff
@@ -84,6 +84,12 @@ void MorningReset()
 		Invoke( "StartMidnightOverlay", timeUntilBlackout );
 	}
 
+	public float GetNormalizeCurrentTime()
```
Comment (randomPoison) [inline Assets/Scripts/Managers/DayCycleManager.cs]: This should be `GetNormalizedCurrentTime()` (you forgot the 'd' in "Normalized"). Also, you should add a comment noting that "normaized" in this case means that it factors out time before the start of the day.

File: Assets/Scripts/UI/InventoryScrollBar.cs [hunk 2]
```diff
@@ -38,7 +38,15 @@ public void NullInventoryBar()
 	void SetIcon( Image image, Sprite icon = null )
 	{
 		image.sprite = icon;
-		image.color = Color.white;
+
+		if( !icon )
+		{
+			image.color = Color.white - (Color)new Vector4( 0f, 0f, 0f, 1f );
```
Comment (randomPoison) [inline Assets/Scripts/UI/InventoryScrollBar.cs]: If you just want to change the alpha of a color we have a `SetAlpha()` extension method for `Color` that would make your intent here more clear.

File: Assets/Scripts/UI/SkillBar.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+	RectTransform _uiBarRectFront = null;
+	RectTransform _uiBarRectBack = null;
+
+	PlayerStats _playerStats = null;
+
+	void Awake()
+	{
+		_uiBarRectBack = GetComponent<RectTransform>();
+		_uiBarRectFront = transform.GetChild(0).GetComponent<RectTransform>();
```
Comment (randomPoison) [inline Assets/Scripts/UI/SkillBar.cs]: You're using a member/property named `transform` here but I don't see where it's coming from. Is this one of the old deprecated automatic properties?

**Comment by randomPoison** [comment]
Assuming everything works correctly this looks good to me.

File: Assets/Scripts/Managers/DayCycleManager.cs (hunk 1 above)
Comment (cjacobwade) [inline Assets/Scripts/Managers/DayCycleManager.cs]: Done

File: Assets/Scripts/UI/InventoryScrollBar.cs (hunk 2 above)
Comment (cjacobwade) [inline Assets/Scripts/UI/InventoryScrollBar.cs]: Done

File: Assets/Scripts/UI/SkillBar.cs (hunk 3 above)
Comment (cjacobwade) [inline Assets/Scripts/UI/SkillBar.cs]: Fixed to use GetComponent<Transform>

**Pull request merged**